stdout = ''
stderr = '''
error: cannot find `x` in scope
  ┌─ tests/fail/elaboration/unification/mismatch/match-equation-output-exprs.fathom:3:8
  │
3 │ match (x : U8) {
  │        ^ unbound name

warning: unreachable pattern
  ┌─ tests/fail/elaboration/unification/mismatch/match-equation-output-exprs.fathom:5:5
  │
5 │     _ => 4 : U64,
  │     ^

error: mismatched types
  ┌─ tests/fail/elaboration/unification/mismatch/match-equation-output-exprs.fathom:5:10
  │
5 │     _ => 4 : U64,
  │          ^^^^^^^ type mismatch, expected `U64`, found `U32`
  │
  = expected `U64`
       found `U32`

warning: unreachable pattern
  ┌─ tests/fail/elaboration/unification/mismatch/match-equation-output-exprs.fathom:6:5
  │
6 │     _ => Type,
  │     ^

error: mismatched types
  ┌─ tests/fail/elaboration/unification/mismatch/match-equation-output-exprs.fathom:6:10
  │
6 │     _ => Type,
  │          ^^^^ type mismatch, expected `Type`, found `U32`
  │
  = expected `Type`
       found `U32`

'''
